doc/admin.md

```diff
@@ -13,11 +13,19 @@ There are, therefore, four YMAL files.
 * `yaml/accounting-server-rc.yaml`      - This configures the replication controller for the APEL Server service
 * `yaml/accounting-server-service.yaml` - This is the APEL server service
 
+## Exposed ports
+
+80   - used by the APEL server service, by the Apache server
+
+443  - used by the APEL server service, by the Apache server (for HTTPS)
+
+3306 - used by the APEL server service and the MySQL service

```

Comment:
slightly, the mysql server and the APEL server both uses this port to communicate with each other. Whereas with 80 and 443 only the Apache server uses it (forwarding requests to the APEL REST interface